website/data/get-involved/4.mdx

```diff
@@ -0,0 +1,5 @@
+## Are you using FDC3?
+
+If you are an existing user of the FDC3 Standard, we would love to hear from you: just email the [FDC3 General List](fdc3@finos.org) with details about how you are using it. 
```

Comment:
Does this email need a `mailto:`? Same with the "contact us" below.

Reply:
yes - it fails the docusaurus link check otherwise